### ISSUE TITLE
Object Navigation: Number included when Reviewing the State of Checkboxes

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#10681]
### Steps to reproduce:

1. Find any checkbox from within Windows, and ensure that the Windows cursor is focused on it

2. Use the NVDA gesture to report the current object

### Actual behavior:

When reporting the current checkbox that has focus, NVDA also reports the number "0" after it - this is also included if choosing to copy the name of the object to the clipboard

### Expected behavior:

In this case, the "0" shouldn't be included as part of NVDA's output when reviewing the object

**Comment by Adriani90** [comment]
cc: @josep

**Comment by CyrilleB79** [comment]
Unless I misunderstood the issue, I cannot reproduce it. Here is my config: Windows 10 1809, NVDA 2019.3beta1 installed. I have tried in Windows Explorer with the "Hidden" checkbox in a file properties dialog.

@elliott94 could you please fill all the fields of the issue template to have more details?

**Comment by Adriani90** [comment]
@elliott94 did you try to reproduce this iwth all addons disabled?

**Comment by Adriani90** [comment]
I can not reproduce it. I tried with object navigation as well as with nvda+shift+o in laptop layout (report current object) or nvda+tab (also report current object). Numbers are not read. Maybe the number is visually displayed, that's why NVDA pronounces it.
I am closing this issue as works for me.
If you have this issue and have exact stepts to reproduce, please comment on it and we can reopen it. Thanks.
